mod/wikirate_source/spec/set/type/source/source_spec.rb

```diff
@@ -179,8 +179,8 @@
       end
     end
   end
-  describe "while rendering views" do 
-    before do 
+  describe "while rendering views" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.